lib/workers/repository/update/branch/schedule.ts

```diff
@@ -25,6 +26,10 @@ function parseCron(
   timezone?: string,
 ): CronExpression | undefined {
   try {
+    const cronScheduleSummary = cronstrue.toString(scheduleText, {
+      throwExceptionOnParseError: false,
+    });
+    logger.debug(`Human-readable summary for cron:: ${cronScheduleSummary}`);
```

Comment:
this causes a weird debug message for all later based strings as we try to parse as cron first 😕

Reply:
@rarkins we should refactor to only log if it's a cron expression and not a later expression.

Reply:
this message is also logged multiple times (for each branch)